include/boost/geometry/algorithms/detail/overlay/get_turn_info_la.hpp

```diff
@@ -529,7 +531,9 @@ struct get_turn_info_linear_areal
                                               bool is_p_last, bool /*is_q_last*/,
                                               OutIt out)
     {
-        bool is_p_spike = ( Version == append_touches ?
+        static const bool is_version_touches = (Version == append_touches);
```

Comment:
Hmm, I see why this is done, but this makes the review much more work.
I would have preferred if these had been a separate commit
Anyway, no need to change now